packages/editor/src/store/selectors.js

```diff
@@ -360,6 +361,20 @@ export function isEditedPostSaveable( state ) {
 	);
 }
 
+/**
```

Comment:
One of the test failures is due to the fact that documentation needs to be regenerated by `npm run docs:build`.